HeatStroke/Services/OBJ/OBJFile.h

```diff
@@ -27,6 +27,17 @@ namespace HeatStroke
 			unsigned int m_uiPositionIndex;
 			unsigned int m_uiUVIndex;
 			unsigned int m_uiNormalIndex;
+
+			// Required to store in a map
+			bool operator<(const OBJVertex& other) const
```

Comment:
I don't like overloading operators when it's not obvious what that operator does.  For an integer, it's obvious what < means.  For a vertex, it's not.  Could we use std::unordered_map to to avoid the need for overloading?